validate.py

```diff
@@ -85,6 +86,34 @@ def get_reviews():
 def get_users():
   return list(sorted(os.listdir('players/')))
 
+def get_user_points():
+  points = {}
+
+  for user in get_users():
+    points[user] = 0
+
+    with open(os.path.join('players', user)) as inf:
+      try:
+        points[user] += int(inf.read())
+      except ValueError:
+        pass
+
+  cmd = ['git', 'log', 'master', '--first-parent', '--format=%s']
+  completed_process = subprocess.run(cmd, stdout=subprocess.PIPE)
+  if completed_process.returncode != 0:
+    raise Exception(completed_process)
+  process_output = completed_process.stdout.decode('utf-8')
+
+  merge_regexp = '^Merge pull request #[\\d]* from ([^/]*)/'
+  for commit_subject in process_output.split('\n'):
+    match = re.match(merge_regexp, commit_subject)
+    if match:
+      commit_username, = match.groups()
+      if commit_username in points:
+        points[commit_username] += 1
```

Comment:
This is a problem. Not Jeff having 43 Itchy'n'Scratchy Fun Bucks - but the fact that Travis currently doesn't appear to *ever* run `validate.py`, because `validate-on-master.sh` will always fail the master-branch check when there aren't enough PR approvals.

⚠️ We don't actually know what effect *any* proposed PR will have in Travis. ⚠️

Imo, that should be the highest priority fix right now, above any other rule changes.

Reply:
@jeffkaufman was this an intended part of the design?

Reply:
> It seems like this will give Jeff 43 points for previous commits

As proposed, this gives me 6 points, not 43:

```
[points-for-changes] $ TRAVIS_PULL_REQUEST=false TRAVIS_REPO_SLUG=jeffkaufman/nomic python3 validate.py
Points:
  TheJhyde: 1
  dchudz: 1
  guoruibiao: 0
  jeffkaufman: 6
  jmitchell: 0
  pavellishin: 0
  sligocki: 0
  tnelling: 0
The game continues.
```

This is because it only counts merge commits.

> I suggest only commit after this one (or some date) count for points.

Sure, that's fine, updated to only count for PRs numbered higher than this one:  cc8a04f

> We don't actually know what effect any proposed PR will have in Travis until everyone approves it.

People can run travis locally, but you're right that this is pretty annoying.  I just wrote a little script that wraps validate.py by pretending to be Travis: #38.  This should make it easier for people to see the effects of proposed changes.